packages/flutter/test_private/bin/test_private.dart

```diff
@@ -225,7 +225,7 @@ class TestCase {
     for (final File test in tests) {
       final String testPath = path.join(path.dirname(test.path), 'lib', path.basenameWithoutExtension(test.path));
       final ProcessRunnerResult result = await runner.runProcess(
-        <String>[flutter, 'test', '--enable-experiment=non-nullable', '--null-assertions', testPath],
+        <String>[flutter, 'test', '--enable-experiment=non-nullable', testPath],
```

Comment:
Is that non-nullable experiment flag used anymore? If this is the Dart SDK experiment flag it is no longer needed.